test/money.html

```diff
@@ -84,20 +83,42 @@
 		});
 
 		test('renderAmount ', () => {
-			assert.equal(renderAmount({ amount }), undefined);
-			assert.isObject(locAmount, 'testamount is configured for locale');
-			testCurrencies.forEach(currency => {
-				const output = locAmount[alt ? currency + '2' : currency] || locAmount[currency];
-				assert.isString(output, `testcurrency is configured for testamount - ${alt} ${isEdge} ${isSafari}`);
-				assert.equal(renderAmount({ amount, currency }), output);
-			});
 			assert.equal(renderAmount(null), undefined);
 			assert.equal(renderAmount(), undefined);
-			assert.equal(renderAmount({ amount, currency: 'USD' }), renderMoney({ amount, currency: 'USD' }));
+			assert.equal(typeof renderAmount({ amount: 100, currency: 'USD'}), 'string');
```

Comment:
https://www.chaijs.com/api/assert/#method_isstring

Reply:
Changed to `isString` on both locations.